packages/react-scripts/config/paths.js

```diff
@@ -48,6 +48,10 @@ function getServedPath(appPackageJson) {
   return ensureSlash(servedUrl, true);
 }
 
+const sWPrecacheImportScript = fs.existsSync(resolveApp('public/service-worker-import.js'))
+  ? 'service-worker-import.js'
```

Comment:
This is an abbreviation and as such I thought `sWPrecacheImportScript` should be fine. Are you suggesting `sWprecacheImportScript `?